package.json

```diff
@@ -2,21 +2,24 @@
   "name": "@ebay/ebayui-core",
   "version": "0.1.0-1",
   "scripts": {
+    "installMarkoV3": "yarn add marko@^3 marko-widgets@^6 -D",
+    "installMarkoV4": "yarn add marko@^4 marko-widgets@^7 -D",
     "prepublishOnly": "node scripts/prepublish",
     "postpublish": "node scripts/postpublish",
     "start": "yarn clean; browser-refresh demo/",
     "coverage": "nyc marko test && node scripts/coverage",
     "report": "nyc report --reporter=html",
-    "test": "yarn coverage && yarn report && yarn test:integration",
+    "test": "yarn coverage && yarn report",
     "test:server": "marko test --server",
     "test:browser": "marko test --browser",
     "test:fps": "karma start",
-    "test:integration": "mocha integration/index.js",
-    "build": "yarn clean; yarn lint && yarn test",
+    "test:integration": "yarn installMarkoV3 && mocha integration && yarn installMarkoV4 && mocha integration && yarn installMarkoV3 && yarn clean:integration",
     "clean": "rimraf .coverage .karma .nyc_output demo/static",
+    "clean:integration": "rimraf integration/*.marko.js integration/static",
     "lint": "yarn lint:less && yarn lint:js",
     "lint:less": "stylelint '**/**.less'",
     "lint:js": "eslint .",
+    "build": "yarn lint && yarn test && mocha integration && yarn clean:integration && yarn installMarkoV4 && nyc marko test && mocha integration",
```

Comment:
I'm having a little trouble following this line. I don't see a reference to installMarkoV3, which I was expecting. Is it not possible to replace a bunch of this stuff with just `yarn test:integration`?

Reply:
The project uses v3 internally by default, so this is assuming that v3 is installed (which is the case most of the time, but especially so in the CI build). We can run `installMarkoV3` here, but it takes additional time and is basically a no-op. I tried to use `build` as an optimized task that's specifically meant for CI.

Alternatively, I set up `test:integration` as something that can run locally, regardless of current installation. That's why that case follows this:
- install v3
- run integration
- install v4
- run integration
- restore v3

The setup is somewhat convoluted, but it's because the underlying scenarios are complicated.

Reply:
Ok, that makes sense. Thanks. I think it would be a good idea to briefly document this on the CONTRIBUTING.md, under the CI section perhaps.